Libraries/LibWeb/Loader/LoadRequest.h

```diff
@@ -41,6 +41,9 @@ class WEB_API LoadRequest {
     ByteBuffer const& body() const { return m_body; }
     void set_body(ByteBuffer body) { m_body = move(body); }
 
+    bool respects_set_cookie() const { return m_respects_set_cookie; }
+    void set_respects_set_cookie(bool respects_set_cookie) { m_respects_set_cookie = respects_set_cookie; }
```

Comment:
I am not a big fan of the name on these method I would rather see them named like below. But let one of the maintainers voice their opinion on this before you change them. (same goes of course for the member variable)
```cpp
bool store_set_cookie_headers() const { return m_store_set_cookie_headers; }
void set_store_set_cookie_headers(bool store_set_cookie_headers) { m_store_set_cookie_headers = store_set_cookie_headers; }
```